build-system/tasks/karma.conf.js

```diff
@@ -123,7 +123,7 @@ module.exports = {
   singleRun: true,
   browserDisconnectTimeout: 10000,
   browserDisconnectTolerance: 2,
-  browserNoActivityTimeout: 4 * 60 * 1000,
+  browserNoActivityTimeout: 400 * 60 * 1000,
```

Comment:
Is this desired change too?

Reply:
Also a leftover from testing. Sorry, I really need to check more thoroughly before submitting PRs in the future.